mmv1/products/cloudidentity/GroupMembership.yaml

```diff
@@ -37,6 +39,8 @@ timeouts:
   update_minutes: 20
   delete_minutes: 20
 custom_code:
+  extra_schema_entry: templates/terraform/extra_schema_entry/cloud_identity_group_membership.tmpl
```

Comment:
Instead of extra_schema_entry I believe you can use a virtual_field. See https://googlecloudplatform.github.io/magic-modules/develop/client-side-fields/